src/js/App/RootApp/DefaultLayout.js

```diff
@@ -95,6 +96,30 @@ const ShieldedRoot = memo(
       >
         <div ref={insightsContentRef} className={classnames('chr-render')}>
           {isGlobalFilterEnabled && <GlobalFilter key={getUrl('bundle')} />}
+          {isProd() && isBeta() && (
+            <Banner variant="info">
+              <Bullseye>
+                <Popover
+                  bodyContent={
+                    <Fragment>
+                      <p>{intl.formatMessage(messages.betaBannerPopover)}</p>
+                      <br />
+                      <a
+                        target="_blank"
+                        href="https://access.redhat.com/support/policy/updates/red-hat-insights/lifecycle"
+                        rel="noreferrer"
+                        style={{ whiteSpace: 'nowrap' }}
+                      >
+                        {intl.formatMessage(messages.learnMoreAboutBeta)} <ExternalLinkAltIcon />
+                      </a>
+                    </Fragment>
+                  }
+                >
+                  <a>Beta</a>
```

Comment:
I know this is true to mocks but, I wonder if we should the Beta term. In application services, the banner says `Service Preview`. Maybe we should stick with that? Feel free to ping the UX/Content person doing the mocks.

Reply:
I have removed the banner from this PR until we'll receive UXD feedback.